extensions/wikia/AdEngine/js/lookup/prebid/prebidHelper.js

```diff
@@ -0,0 +1,34 @@
+define('ext.wikia.adEngine.lookup.prebid.prebidHelper', function() {
+	var adUnits = [];
+
+	function getAdUnits(adapter, slots) {
+		var adaptersAdUnits = [];
+
+		Object.keys(slots).forEach(function(slotName) {
+			adaptersAdUnits.push(adapter.prepareAdUnit(slotName, slots[slotName]));
+		});
+
+		return adaptersAdUnits;
+	}
+
+	function addAdUnits(adapterAdUnits) {
+		adapterAdUnits.forEach(function (adUnit) {
+			adUnits.push(adUnit);
+		});
+	}
+
+	function setupAdUnits(adapters, skin) {
+		adapters.forEach(function (adapter) {
+			if (adapter && adapter.isEnabled()) {
+				addAdUnits(getAdUnits(adapter, adapter.getSlots(skin)));
+			}
+		});
+
+		return adUnits;
+	}
+
+	return {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)